modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -997,7 +996,22 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
    * @param bitgo
    */
   static async getBitGoChallenges(bitgo: BitGoBase): Promise<GetBitGoChallengesApi> {
-    return await bitgo.get(bitgo.url('/tss/ecdsa/challenges', 2)).send().result();
+    const res = await bitgo.get(bitgo.url('/tss/ecdsa/challenges', 2)).send().result();
+    if (
+      !res.bitgoNitroHsm ||
+      !res.bitgoNitroHsm.ntilde ||
+      !res.bitgoNitroHsm.h1 ||
+      !res.bitgoNitroHsm.h2 ||
+      !res.bitgoNitroHsm.ntildeProof ||
+      !res.bitgoInstitutionalHsm ||
+      !res.bitgoInstitutionalHsm.ntilde ||
+      !res.bitgoInstitutionalHsm.h1 ||
+      !res.bitgoInstitutionalHsm.h2 ||
+      !res.bitgoInstitutionalHsm.ntildeProof
+    ) {
+      throw new Error('Expected BitGo challenge proof to be present. Contact support@bitgo.com.');
```

Comment:
This was the existing error message, but instead of being thrown downstream, its thrown at the time we fetch the data. This allows downstream usage to have better types.